### PR TITLE
chore(tabs): update to UDL styles

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Update Tabs styles
- [x] Update anatomy table on Tabs doc page

### CODE REVIEW

File: packages/paste-core/components/tabs/src/Tab.tsx [hunk 1]
```diff
@@ -128,6 +128,7 @@ const Tab = React.forwardRef<HTMLDivElement, TabProps>(({children, ...tabProps},
               {...textStyles}
               fontSize="fontSize30"
               fontWeight="fontWeightSemibold"
+              lineHeight="lineHeight20"
```
Comment (SiTaggart) [inline packages/paste-core/components/tabs/src/Tab.tsx:131]: Can you remove this? The 20px line height will come by default when we merge in https://github.com/twilio-labs/paste/pull/656.

That way we keep the line heights in components in line with their font sizes

File: packages/paste-core/components/tabs/src/Tab.tsx [hunk 2]
```diff
@@ -78,26 +86,6 @@ const getTabBoxStyles = (orientation: Orientation, variant: Variants): {} => {
   }
 };
 
-const getTabTextStyles = (props: TabProps): {} => {
```
Comment (TheSisb) [inline packages/paste-core/components/tabs/src/Tab.tsx:81]: Nice, thanks!

File: packages/paste-core/components/tabs/src/Tab.tsx [hunk 3]
```diff
@@ -111,21 +99,20 @@ const Tab = React.forwardRef<HTMLDivElement, TabProps>(({children, ...tabProps},
...
             cursor={props['aria-disabled'] ? 'not-allowed' : 'pointer'}
-            transition="border-color 100ms ease"
-            textOverflow={tab.orientation !== 'vertical' ? 'ellipsis' : undefined}
             overflow={tab.orientation !== 'vertical' ? 'hidden' : undefined}
+            position="relative"
+            textOverflow={tab.orientation !== 'vertical' ? 'ellipsis' : undefined}
+            transition="border-color 100ms ease"
             whiteSpace={tab.orientation !== 'vertical' ? 'nowrap' : undefined}
           >
             <Text
```
Comment (TheSisb) [inline packages/paste-core/components/tabs/src/Tab.tsx:113]: Should we remove this `Text` component and put the styles on the wrapping `Box`?

Comment (richbachman) [reply packages/paste-core/components/tabs/src/Tab.tsx:113]: Good call. Those styles can all go on `Box`.

**Pull request merged**